examples/spark/spark-gluster/spark-worker-controller.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: spark-worker
-          image: gcr.io/google_containers/spark-worker:1.5.1_v2
+          image: gcr.io/google_containers/spark-worker:1.5.2_v1
```

Comment:
this needs to be changed to spark:1.5.2_v1 and the command: /start-worker needs to be added